docs/documentation/make-first-prototype/use-components.md

```diff
@@ -71,6 +71,11 @@ Your component code should now look like this:
 
 Your page should now look like this:
 
-![Screenshot of the question page with radios](/public/images/docs/tutorial-radios.png)
+<figure>
+
+![](/public/images/docs/tutorial-radios.png)
```

Comment:
suggest: `Web page with the heading "How many balls can you juggle?", 3 radios and a continue button`